apps/openmw/mwmechanics/actors.cpp

```diff
@@ -782,9 +786,14 @@ namespace MWMechanics
                             isDamageEffect = true;
                     }
 
-                    MWWorld::Ptr caster = MWBase::Environment::get().getWorld()->searchPtrViaActorId(spell.mCasterActorId);
-                    if (isDamageEffect && caster == player)
-                        killedByPlayer = true;
+                    if (isDamageEffect)
+                    {
+                        if (caster == player || playerFollowers.find(caster) != playerFollowers.end())
+                        {
+                            killedByPlayer = true;
+                            break;
+                        }
+                    }
                 }
             }
             if (killedByPlayer)
```

Comment:
Not sure if we really need to increase werewolf kills here since werewolves can not cast spells.

Reply:
There's the case when the player could have transformed into a werewolf while the magic effect that killed the NPC was still in action, I think.